examples/Kittens/Sample/BlurbNode.m

```diff
@@ -76,14 +76,14 @@ - (void)didLoad
 #if UseAutomaticLayout
 - (ASLayoutSpec *)layoutSpecThatFits:(ASSizeRange)constrainedSize
 {
-  return
-  [ASInsetLayoutSpec
-   newWithInsets:UIEdgeInsetsMake(kTextPadding, kTextPadding, kTextPadding, kTextPadding)
-   child:
-   [ASCenterLayoutSpec
-    newWithCenteringOptions:ASCenterLayoutSpecCenteringX // Center the text horizontally
-    sizingOptions:ASCenterLayoutSpecSizingOptionMinimumY // Takes up minimum height
-    child:_textNode]];
+  ASCenterLayoutSpec *centerSpec = [[ASCenterLayoutSpec alloc] init];
+  centerSpec.centeringOptions = ASCenterLayoutSpecCenteringX;
+  centerSpec.sizingOptions = ASCenterLayoutSpecSizingOptionMinimumY;
+  centerSpec.child = _textNode;
+  
+  
+  return [ASInsetLayoutSpec insetLayoutSpecWithInsets:UIEdgeInsetsMake(kTextPadding, kTextPadding, kTextPadding, kTextPadding)
```

Comment:
This would probably look nicer with a UIEdgeInsets insets = ..., and then a single line method call below that.